Makefile

```diff
@@ -3,6 +3,7 @@ SHELL := /bin/bash
 MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 GO ?= go
+KUADRANT_NAMESPACE=kuadrant-system
```

Comment:
why add this make var?

Reply:
No reason - it probably was left over from a previous commit this branch didn't have when created, seen at 4156da9 . I'll go ahead and remove... good catch again! :)